.gitignore

```diff
@@ -2,3 +2,6 @@
 tmp
 .bundle
 vendor
+Gemfile.lock
```

Comment:
Not to mention Travis breaking after things like `git cherry-pick` / rebasing like here: https://travis-ci.org/grosser/parallel_tests/jobs/66961221#L97